questions.md

```diff
@@ -0,0 +1,8 @@
+#Additional Questions
+
+* Here is where I can list any additional questions that I have
+
+1. I'm wondering what problems I may run into in the classroom
+2. What will I need to train my students on
+3. Will they enjoy this
```

Comment:
Students report that after 3 months they spend more time on GitHub.com for their own projects than on class projects. If students want to learn tools that will prepare them for industry, GitHub is it.